karma.conf.js

```diff
@@ -0,0 +1,82 @@
+var path = require('path');
+
+module.exports = function(karma) {
+  'use strict';
+
+  karma.set({
+    basePath: __dirname,
+
+    frameworks: ['jasmine'],
+
+    files: [
+      { pattern: 'tests.bundle.ts', watched: false }
+    ],
+
+    exclude: [],
+
+    preprocessors: {
+      'tests.bundle.ts': ['coverage', 'webpack', 'sourcemap']
+    },
+
+    reporters: ['mocha', 'coverage'],
+
+    coverageReporter: {
+      dir: 'coverage/',
+      reporters: [
+        { type: 'text-summary' },
+        { type: 'json' },
+        { type: 'html' }
+      ]
+    },
+
+    browsers: ['Chrome'],
+
+    port: 9018,
+    runnerPort: 9101,
+    colors: true,
+    logLevel: karma.LOG_INFO,
+    autoWatch: true,
+    singleRun: false,
+    webpackServer: { noInfo: true },
+    webpack: {
```

Comment:
This is probably been discussed off-line at length somewhere, but I just can't resist asking about it :-)

Is there an opportunity here to use just a single bundler to bundle for the two different purposes (release and unit test)? It probably works okay to drag along to bundlers (and if the project was bigger, to bundling ecosystems) for those purposes, but it feels sort of silly, and feels like it leaves opportunities for things to go wrong.